synapse/federation/transport/server.py

```diff
@@ -138,6 +139,12 @@ def strip_quotes(value):
         logger.info("Request from %s", origin)
         request.authenticated_entity = origin
 
+        # If we get a valid signed request from the other side, its probably
+        # alive
+        retry_timings = yield self.store.get_destination_retry_timings(origin)
+        if retry_timings and retry_timings["retry_last_ts"]:
+            self.store.set_destination_retry_timings(origin, 0, 0)
```

Comment:
is it worth logging this?

Also: you need to either `yield` or `preserve_fn` on this I think